README.md

```diff
@@ -88,30 +88,34 @@ Maximum number of Functions to bundle at the same time.
 
 #### jsBundler
 
-_Type_: `number`\
+_Type_: `string`\
 _Default value_: `zisi`
 
 The bundler to use when processing JavaScript functions. Possible values: `zisi`, `esbuild`, `esbuild_zisi`.
 
 When the value is `esbuild_zisi`, `esbuild` will be used with a fallback to `zisi` in case of an error.
 
-#### jsExternalModules
+#### config
+
+_Type_: `object`\
+_Default value_: `{}`
+
+An object matching glob-like expressions to objects containing configuration properties. Whenever a function name
+matches one of the expressions, it inherits the configuration properties.
 
-_Type_: `array<string>`\
-_Default value_: `[]`
+The following properties are accepted:
 
-List of Node modules to include separately inside a node_modules directory.
+- `jsExternalModules`
```

Comment:
Should we consider renaming those properties (in a separate PR) to match how they are called in `@netlify/config`?

Reply:
> Should we consider renaming those properties (in a separate PR) to match how they are called in `@netlify/config`?

You mean the camel case vs. snake case difference? In https://github.com/netlify/build/pull/2414 I had Build doing the translation. Do you think ZISI should just handle the snake case properties?

Reply:
No, it probably should keep using camelCase.
I meant, the new names suggested here: https://github.com/netlify/build/pull/2390#issuecomment-799535006

Reply:
> No, it probably should keep using camelCase.
> 
> I meant, the new names suggested here: https://github.com/netlify/build/pull/2390#issuecomment-799535006

Ah, yes! Definitely. I was just waiting for a 👍/👎 from @verythorough before making the change.